examples/UKCore-DocumentReference-ECG-Example.xml

```diff
@@ -0,0 +1,43 @@
+<DocumentReference xmlns="http://hl7.org/fhir">
```

Comment:
We have Genomics related example for this, if its necessary to use it: https://github.com/NHSDigital/NHSDigital-FHIR-Genomics-ImplementationGuide/blob/main/DocumentReference/DocumentReference-PharmCAT-Example.json